test/templates/counter/index.njk

```diff
@@ -1,70 +1,35 @@
 {% extends "layout.njk" %}
-{% from "./component.njk" import counter %}
```

Comment:
If we're no longer importing the counter component, then we should remove that file.  Or, use the component file and redo the examples to call that.

Reply:
@greg-a-smith that's what I did, didn't I?

Reply:
You are not importing it, but the file still exists.  To match all the other playground examples, we should actually keep the file, import it and pass the necessary parameters to it.

Reply:
reverted it